### PR TITLE
Adds wrapper for Game::issueCommand

### LINKED ISSUE #47: Wrap Game::issueCommand

### PR DESCRIPTION
**Pull request by kpp**
Closes #47

### CODE REVIEW

File: src/Game.cpp [hunk 1]
```diff
@@ -351,6 +352,18 @@ void Game_setLocalSpeed(Game* self, int speed) {
     reinterpret_cast<BWAPI::Game*>(self)->setLocalSpeed(speed);
 }
 
+bool Game_issueCommand(Game* self, UnitIterator* units, UnitCommand command) {
+    IteratorBase* iter = reinterpret_cast<IteratorBase*>(units);
+    assert(iter->id() == itUnit);
+    BWAPI::Unitset unitset;
+    for (; iter->valid(); iter->next()) {
+        Unit* const unit = reinterpret_cast<Unit*>( iter->get() );
```
Comment (nlinker) [inline src/Game.cpp]: inconsistent style, I'd propose to remove the spacing:
`reinterpret_cast<Unit*>(iter->get());`

File: src/Game.cpp [hunk 2]
```diff
@@ -351,6 +352,18 @@ void Game_setLocalSpeed(Game* self, int speed) {
...
+        assert(unit);
+        unitset.insert(reinterpret_cast<BWAPI::Unit>(unit));
```
Comment (nlinker) [inline src/Game.cpp]: You can use `cast_from_bw(unit)`, it does this `reinterpret_cast` underneath.

File: src/Game.cpp (hunk 1 above)
Comment (nlinker) [inline src/Game.cpp]: Also you can write it like this
```
Unit* const unit = cast_from_bw(iter->get());
assert(unit);
unitset.insert(cast_to_bw(unit));
```

**Review by nlinker — COMMENTED** [review]
Some suggestions

File: src/Game.cpp (hunk 1 above)
Comment (kpp) [inline src/Game.cpp]: done

**Review by nlinker — APPROVED** [review]

**Pull request merged**